contracts/Overtime/SportsAMMV2.sol

```diff
@@ -381,7 +385,19 @@ contract SportsAMMV2 is Initializable, ProxyOwned, ProxyPausable, ProxyReentranc
     /// @notice exercise specific ticket
     /// @param _ticket ticket address
     function exerciseTicket(address _ticket) external nonReentrant notPaused onlyKnownTickets(_ticket) {
-        _exerciseTicket(_ticket);
+        _exerciseTicket(_ticket, address(0), false);
+    }
+
+    /// @notice exercise specific ticket to an off ramp collateral
+    /// @param _ticket ticket address
+    /// @param _exerciseCollateral collateral address to off ramp to
+    /// @param _inEth offramp with ETH
+    function exerciseTicketOffRamp(
+        address _ticket,
+        address _exerciseCollateral,
+        bool _inEth
+    ) external nonReentrant notPaused onlyKnownTickets(_ticket) {
+        _exerciseTicket(_ticket, _exerciseCollateral, _inEth);
```

Comment:
Missing check that only ticket owner can call this method

Reply:
Ok, yea makes sense only the ticketOwner to call offramp
and default exercise by anyone